gpt4all-backend/scripts/gen_tokenizer_include.py

```diff
@@ -0,0 +1,136 @@
+import sys
```

Comment:
Could you add a short readme on how to generate this file for future BPE tokenizers?

Reply:
https://github.com/nomic-ai/gpt4all/pull/661/files?short_path=a9d8106#diff-a9d8106c93dbe95ab455b8037ed74efa894585b30d0753a3fd6500a3ba6d1b9a